pendant/theme.json

```diff
@@ -1,297 +1,310 @@
+					"color": "#000000",
+					"name": "Foreground"
+				},
+				{
+					"slug": "background",
+					"color": "#ffffff",
+					"name": "Background"
+				},
+				{
+					"slug": "tertiary",
+					"color": "#f1ede6",
+					"name": "Tertiary"
+				}
+			],
+			"link": true
+		},
+		"custom": {
+			"gap": {
+				"horizontal": "clamp(20px, 12vw - 40px, 100px)"
+			}
+		},
+		"spacing": {
+			"units": [
+				"%",
+				"px",
+				"em",
+				"rem",
+				"vh",
+				"vw"
+			],
+			"blockGap": true,
+			"margin": true,
+			"padding": true
+		},
+		"typography": {
+			"fontFamilies": [
+				{
+					"fontFamily": "Jost, sans-serif",
+					"slug": "body-font",
+					"name": "Body (System Font)",
+					"fontFace": [
+						{
+							"fontFamily": "Jost",
+							"fontWeight": "300 400",
+							"fontStyle": "normal",
+							"fontStretch": "normal",
+							"src": [
+								"file:./assets/fonts/Jost-Light.ttf"
+							]
+						},
+						{
+							"fontFamily": "Jost",
+							"fontWeight": "500 600",
+							"fontStyle": "normal",
+							"fontStretch": "normal",
+							"src": [
+								"file:./assets/fonts/Jost-Medium.ttf"
+							]
+						}
+					]
+				},
+				{
+					"fontFamily": "'Literata', serif",
+					"slug": "heading-font",
+					"name": "Headings (System Font)",
+					"fontFace": [
+						{
+							"fontFamily": "Literata",
+							"fontWeight": "400",
+							"fontStyle": "normal",
+							"fontStretch": "normal",
+							"src": [
+								"file:./assets/fonts/Literata_72pt-Light.ttf"
+							]
+						},
+						{
+							"fontFamily": "Literata",
+							"fontWeight": "400",
+							"fontStyle": "italic",
+							"fontStretch": "normal",
+							"src": [
+								"file:./assets/fonts/Literata_72pt-LightItalic.ttf"
+							]
+						},
+						{
+							"fontFamily": "Literata",
+							"fontWeight": "800 900",
+							"fontStyle": "normal",
+							"fontStretch": "normal",
+							"src": [
+								"file:./assets/fonts/Literata_72pt-Bold.ttf"
+							]
+						},
+						{
+							"fontFamily": "Literata",
+							"fontWeight": "800 900",
+							"fontStyle": "italic",
+							"fontStretch": "normal",
+							"src": [
+								"file:./assets/fonts/Literata_72pt-BoldItalic.ttf"
+							]
+						}
+					]
+				}
+			],
+			"fontSizes": [
+				{
+					"name": "Small",
+					"size": "1rem",
+					"slug": "small"
+				},
+				{
+					"name": "Medium",
+					"size": "1.25rem",
+					"slug": "medium"
+				},
+				{
+					"name": "Large",
+					"size": "1.5rem",
+					"slug": "large"
+				},
+				{
+					"name": "Extra Large",
+					"size": "2rem",
+					"slug": "x-large"
+				}
+			],
+			"lineHeight": true
+		},
+		"layout": {
+			"contentSize": "820px",
+			"wideSize": "1000px"
+		},
+		"border": {
+			"color": true,
+			"radius": true,
+			"style": true,
+			"width": true
+		}
+	},
+	"styles": {
+		"blocks": {
+			"core/button": {
+				"border": {
+					"radius": "0px"
+				},
+				"color": {
+					"background": "var:preset|color|foreground",
```

Comment:
@pbking @MaggieCabrera would either of you be able to please point me to how/what this syntax is doing please? `var:preset|color|foreground`